expression-language/pom.xml

```diff
@@ -25,6 +25,11 @@
       <artifactId>zeebe-util</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-scheduler</artifactId>
```

Comment:
:question: We need the scheduler here ? :thinking:

Reply:
It uses the `ActorClock` :shrug: Would be nice to abstract that away soon, it only needs a source for the current epoch millis!

Reply:
Ok maybe we just have clock interface somewhere and then just feed the implementation on parser 
creation.

Reply:
I'll create a follow-up issue for this :+1: I didn't find any usable Clock interface ad-hoc but I think we could just create `ZeebeFeelEngineClock` in the engine and pass that into the feel engine constructor.

Reply:
I guess ZeebeClock would be enough, no need to have an interface that is only for that case. We need that in other places as well.

Reply:
https://github.com/camunda/zeebe/issues/9609